### ISSUE TITLE
Help E/W (UAA local) users get their (MFA) access to GovCloud provisioned

### ISSUE THREAD

**Issue by mogul** [18F/cg-product#228]
- explain what is going on (eg MFA setup process)
- send invitations at the time when we're ready for everyone to migrate

**Comment by afeld** [comment]
Blocked by https://github.com/18F/cg-product/issues/138.

**Comment by mogul** [comment]
Probably blocked on https://github.com/18F/cg-product/issues/78 now.

**Comment by mogul** [comment]
I'm thinking maybe we kill this one... There's another story covering explaining what's going on (the sandbox deprecation mail) and that one itself is pending #78 so that we don't generate a bunch of invitations that go stale before people can use them.

**Comment by fureigh** [comment]
[@mogul - Bret Mogilefsky] Including a GovCloud self-invitation link in the email to East/West sandbox users will cover this, yes? Is there anything else that this was meant to address?

**Comment by mogul** [comment]
[@fureigh – Rhys Fureigh] yup, this was written when self-invitations weren't yet a thing and we weren't sure if they'd arrive before we needed to get everyone migrated over. So you can feel free to archive/close/whatever-makes-sense-for-how-you're-tracking-progress-in-Favro... Meanwhile I'll close the GitHub issue.

**Comment by fureigh** [comment]
Marking this `wontfix` per @mogul's comment.
